### ISSUE TITLE
Spyder raises error whenever executing commands via console

### ISSUE THREAD

**Issue by EdDanileyko** [spyder-ide/spyder#7203]
## Description
Not sure what is the issue as I am not doing anything really out of the ordinary, but every time I attempt to do anything in the console, this error pops up. started after checking len() of a linked list node instance. Output is as expected, but this error is raised upon every command executed.

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "C:\Program Files (x86)\Microsoft Visual Studio\Shared\Anaconda3_64\lib\site-packages\qtconsole\base_frontend_mixin.py", line 163, in _dispatch
    handler(msg)
  File "C:\Program Files (x86)\Microsoft Visual Studio\Shared\Anaconda3_64\lib\site-packages\spyder\widgets\ipythonconsole\namespacebrowser.py", line 220, in _handle_execute_reply
    self.handle_exec_method(msg)
  File "C:\Program Files (x86)\Microsoft Visual Studio\Shared\Anaconda3_64\lib\site-packages\spyder\widgets\ipythonconsole\shell.py", line 351, in handle_exec_method
    view = ast.literal_eval(literal)
  File "C:\Program Files (x86)\Microsoft Visual Studio\Shared\Anaconda3_64\lib\ast.py", line 48, in literal_eval
    node_or_string = parse(node_or_string, mode='eval')
  File "C:\Program Files (x86)\Microsoft Visual Studio\Shared\Anaconda3_64\lib\ast.py", line 35, in parse
    return compile(source, filename, mode, PyCF_ONLY_AST)
  File "<unknown>", line 1
    {'a': {'type': 'ListNode', 'size': 1, 'color': '#800000', 'view': <__main__.ListNode object at 0x000001E5A8C567B8>}}
                                                                      ^
SyntaxError: invalid syntax

## Version and main components

* Spyder Version: 3.2.3 
* Python Version: 3.6.2
* Qt Versions: 5.6.2, PyQt5 5.6 on Windows

## Dependencies
```
pyflakes >=0.6.0 :  1.5.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.3 (OK)
numpy >=1.7      :  1.13.1 (OK)
sphinx >=0.6.6   :  1.6.3 (OK)
rope >=0.9.4     :  0.10.5 (OK)
jedi >=0.9.0     :  0.10.2 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
sympy >=0.7.3    :  1.1.1 (OK)
cython >=0.21    :  0.26.1 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
IPython >=4.0    :  6.1.0 (OK)
pylint >=0.25    :  1.7.2 (OK)

```

**Comment by ccordoba12** [comment]
This is fixed in our latest version (3.2.8). Please update by opening the Anaconda Prompt and running there

    conda update spyder

**Comment by CAM-Gerlach** [comment]
FYI, this error appears to be a duplicate of #5636 , which is fixed in Spyder 3.2.6.
